### PR TITLE
deploy synthetic probe endpoint

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: cluster/manifests/skipper/routegroup-probe.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: zalando.org/v1
+kind: RouteGroup
+metadata:
+  name: synthetic-probe
+  namespace: default
```
Comment (hjacobs) [inline cluster/manifests/skipper/routegroup-probe.yaml]: Maybe use a different namespace, e.g. `kube-system`?

Comment (hjacobs) [reply cluster/manifests/skipper/routegroup-probe.yaml]: Not that users remove this RouteGroup because it's in `default`.

Comment (szuecs) [reply cluster/manifests/skipper/routegroup-probe.yaml]: Moved to kube-system

**Comment by szuecs** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**